core/src/main/scala/kafka/server/KafkaServer.scala

```diff
@@ -421,7 +453,7 @@ class KafkaServer(val config: KafkaConfig, time: Time = SystemTime, threadNamePr
               case ioe: IOException =>
                 ioException = true
                 warn("Error during controlled shutdown, possibly because leader movement took longer than the configured socket.timeout.ms: %s".format(ioe.getMessage))
-                // ignore and try again
+              // ignore and try again
```

Comment:
This change looks wrong. A few other examples. I'd try to avoid reformatting code outside of the code you are changing (unless the existing formatting is wrong).